Description: accept newer minor version of crate egg
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2026-04-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/crates/wasm-mutate/Cargo.toml
+++ b/crates/wasm-mutate/Cargo.toml
@@ -12,7 +12,7 @@
 
 [dependencies]
 clap = { workspace = true, optional = true }
-egg = "0.6.0"
+egg = ">= 0.6.0, <= 0.7"
 log = { workspace = true }
 rand = { workspace = true }
 thiserror = { workspace = true }
